### ISSUE TITLE
Collapse bottom bar on scroll does not aplly after closing settings.

### ISSUE THREAD

**Issue by Niko4u5** [Bnyro/ConnectYou#133]
### Steps to reproduce

1. open setings
2. select Collapse bottom bar on scroll
3. close setings
4. scroll your contacts

### Expected behavior

bottom bar should collaps

### Actual behavior

only after switching between device and local does the change apply

### Connect You version

3.0 (17)

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
